code/modules/clothing/spacesuits/alien_suits.dm

```diff
@@ -143,30 +143,47 @@
 	desc = "An almost organic looking nonhuman pressure suit."
 
 /obj/item/clothing/under/vox
-	has_sensor = FALSE
-	icon = 'icons/obj/clothing/species/vox/uniforms.dmi'
+	icon = 'icons/mob/clothing/species/vox/under/misc.dmi'
 	species_restricted = list("Vox")
 	sprite_sheets = list(
 		"Vox" = 'icons/mob/clothing/species/vox/under/misc.dmi'
 		)
 
 /obj/item/clothing/under/vox/vox_casual
-	name = "alien clothing"
-	desc = "This doesn't look very comfortable."
-	icon_state = "vox-casual-1"
-	item_color = "vox-casual-1"
-	item_state = "vox-casual-1"
-	body_parts_covered = LEGS
-
-/obj/item/clothing/under/vox/vox_robes
-	name = "alien robes"
-	desc = "Weird and flowing!"
-	icon_state = "vox-casual-2"
-	item_color = "vox-casual-2"
-	item_state = "vox-casual-2"
+	name = "Vox jumpsuit"
+	desc = "These loose clothes are optimized for the labors of the lower castes aboard the arkships. Large openings in the top allow for breathability while the pants are durable yet flexible enough to not restrict movement."
+	icon_state = "vox-jumpsuit_s"
+	item_color = "vox-jumpsuit"
+	item_state = "vox-jumpsuit"
+
+/obj/item/clothing/suit/hooded/vox_robes
+	name = "Vox robes"
+	desc = "Large, comfortable robes worn by those who need a bit more covering. The thick fabric contains a pocket suitable for those that need their hands free during their work, while the cloth serves to cover scars or other injuries to the wearer's body."
+	icon = 'icons/mob/clothing/species/vox/suit.dmi'
+	icon_state = "vox-robes"
+	item_color = "vox-robes"
+	item_state = "vox-robes"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	hoodtype = /obj/item/clothing/head/hooded/vox_robe_hood
+	species_restricted = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi'
+	)
+
+//Vox Robes Hood
+/obj/item/clothing/head/hooded/vox_robe_hood
+	name = "Vox hood"
+	desc = "The thick fabric of this hood serves a variety of purposes to the vox wearing it - serving as a method to hide a scarred face or a way to keep warm in the coldest areas onboard the ship."
```

Comment:
The correct term is "onboard" here?

Reply:
Yes. It was a bit confusing, but aboard is referring to the act of actively boarding something, but onboard is when something is already on or in a vehicle. "Come aboard" versus "Onboard motor". I have adjusted descriptions to remedy the inconsistencies to be "onboard".